packages/cli-platform-android/src/commands/runAndroid/listAndroidDevices.ts

```diff
@@ -0,0 +1,126 @@
+import {execSync} from 'child_process';
+import adb from './adb';
+import getAdbPath from './getAdbPath';
+import {getEmulators} from './tryLaunchEmulator';
+import os from 'os';
+import prompts from 'prompts';
+import chalk from 'chalk';
+import {CLIError} from '@react-native-community/cli-tools';
+
+type DeviceData = {
+  deviceId: string | undefined;
+  readableName: string;
+  connected: boolean;
+  type: 'emulator' | 'phone';
+};
+
+function toPascalCase(value: string) {
+  return value !== '' ? value[0].toUpperCase() + value.slice(1) : value;
+}
+
+/**
+ *
+ * @param deviceId string
+ * @returns name of Android emulator
+ */
+function getEmulatorName(deviceId: string) {
+  // maybe close it with trycatch block?
+  const adbPath = getAdbPath();
+  const buffer = execSync(`${adbPath} -s ${deviceId} emu avd name`);
```

Comment:
why not `execa`?

Reply:
No reason actually ;)
Guess, execSync was already imported and used in other places so went same way

Reply:
Ok, we can keep it for consistency now